.github/workflows/test.yml

```diff
@@ -37,7 +37,10 @@ jobs:
 
       # Test setup latest version
       - uses: ./
-      - run: sam --version | grep -Fv 1.18.2
+      - run: |
+          version=$(curl https://pypi.org/pypi/aws-sam-cli/json | jq -r .info.version)
+          sam --version | grep -F "$version"
+        shell: bash
```

Comment:
Is there a chance this test becomes a flanky test if SAM CLI updates after SAM CLI is installed and we release a new version? 😄

Reply:
If you can update SAM CLI between when `uses` and `run` run, then yes. I'd congratulate you for such a feat.